shared/js/background/utils.js

```diff
@@ -413,3 +413,38 @@ export async function reloadCurrentTab () {
         browser.tabs.reload(tab.id)
     }
 }
+
+const dayMultiplier = 24 * 60 * 60 * 1000
+
+/**
+ * Converts ATB value into date
+ * @param {string} atb
+ * @returns {number|null}
+ */
+export function getInstallTimestamp (atb) {
+    const match = atb.match(/^v?(\d+)-(\d)(.+)?$/i)
+    if (!match) return null
+
+    const startDate = 1456272000000
+    const weeksSince = (parseInt(match[1], 10) - 1) * 7 * dayMultiplier
+    const daysSince = (parseInt(match[2], 10) - 1) * dayMultiplier
+    const installTimestamp = new Date(startDate + weeksSince + daysSince).getTime()
+    return isNaN(installTimestamp) ? null : installTimestamp
+}
+
+/**
+ * Checks if the extension was installed recently
+ * @param {number} daysCount
+ * @returns {boolean}
+ */
+export function isInstalledWithinDays (daysCount) {
```

Comment:
@alistairjcbrown as discussed it's worth adding a couple of unit tests for this - I'd just pass in the `Date.now()` as a default param too :)